index.js

```diff
@@ -99,6 +100,15 @@ module.exports = function (text, opts) {
 
 	var NL = '\n';
 	var PAD = ' ';
+
+	if (opts.align) {
+		text = ansiAlign(text, {
+			align: opts.align,
+			split: NL,
+			pad: PAD
```

Comment:
@sindresorhus True, I was just being paranoid about depending on that assumption, in case the defaults for `ansi-align` ever change.

Since that seems unlikely, I'll go ahead and change this.